test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] You have correctly added the test.md file 👍, however as a reminder for this project you should add a .gitignore and add test.md and node_modules inside it. this will ensure that those files are ignored and not tracked by git when adding commits.